src/components/Card.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { SyntheticEvent, useMemo } from 'react';
+import { IconButton } from 'react-file-utils';
+import { sanitizeURL } from '../utils';
+
+import { AvatarIcon, CloseIcon } from './Icons';
+
+// export type Props = {|
+// 	image?: ?string,
+// 	images?: Array<{ image: string }>,
+// 	alt?: ?string,
+// 	title?: ?string,
+// 	url?: string,
+// 	nolink?: boolean,
+// 	description?: ?string,
+// 	handleClose?: (e: SyntheticEvent<>) => mixed,
+//  |};
+
+type CardProps = {
+  alt?: string;
+  description?: string;
+  handleClose?: (e: SyntheticEvent) => void;
+  image?: string | null;
+  images?: Array<{ image: string }>;
+  nolink?: boolean;
+  title?: string;
+  url?: string;
+};
+
+const trimUrl = (url?: string) =>
+  url
+    ?.replace(/^(?:https?:\/\/)?(?:www\.)?/i, '')
+    .split('/')
+    .shift();
+
+export const Card = ({
+  alt,
+  images = [],
+  image: imageURL, // = images.length ? images[0].image : undefined,
+  handleClose,
+  description,
+  nolink,
+  url,
+  title,
+}: CardProps) => {
+  const sanitizedURL = useMemo(() => sanitizeURL(url), [url]) as string | undefined;
```

Comment:
instead of cast let's put a  //@ts-expect-error so that we won't forget to fix it once utils is in TS

Reply:
`sanitizeURL` returns `any` and `trimURL` accepts it without any error. Adding //@ts-expect-error makes TS yell at me that the directive is unnecessary.

Reply:
then a TODO will do the job so we won't forget to fix it